pkg/portal/middleware/log.go

```diff
@@ -115,6 +118,13 @@ func Log(env env.Core, auditLog, baseLog *logrus.Entry) func(http.Handler) http.
 				resultType := audit.ResultTypeSuccess
 				if statusCode >= http.StatusBadRequest {
 					resultType = audit.ResultTypeFail
+					otelAuditMsg.Record.OperationResult = msgs.Failure
```

Comment:
This seems duplicated from `frontend/middleware/log.go` - why? Can we DRY it out? Would we ever want these two to differ?

Reply:
Pretty much everything in this file, yes.

Reply:
(https://www.diffchecker.com/CnmjpW5z/)